### PR TITLE
IGVF-678-seqspec-to-sequence-file

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/sequence_file.json [hunk 1]
```diff
@@ -160,6 +160,12 @@
                 "I1",
                 "I2"
             ]
+        },
+        "seqspec": {
+            "title": "Link to seqspec YAML File",
+            "description": "The link to the associated seqspec YAML ConfigurationFile object.",
```
Comment (gabdank) [inline src/igvfd/schemas/sequence_file.json:166]: please change description to be: The link to the associated seqspec YAML configuration file."

File: src/igvfd/loadxl.py [hunk 2]
```diff
@@ -51,10 +51,10 @@
     'auxiliary_set',
     'prediction',
     'reference_file',
+    'configuration_file',
```
Comment (gabdank) [inline src/igvfd/loadxl.py:54]: if the goal is to load fist seqspec before sequencing file then the proper way to do it is pipeline 1 pipeline 2 below.

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/sequence_file.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/sequence_file.json:166]: changed

File: src/igvfd/loadxl.py (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/loadxl.py]: fixed

File: src/igvfd/tests/data/inserts/sequence_file.json [hunk 3]
```diff
@@ -16,7 +16,8 @@
         "upload_status": "pending",
         "file_set": "igvf:basic_analysis_set",
         "sequencing_run": 12,
-        "sequencing_platform": "/platform-terms/747dc8bf-9e8c-3dbc-967f-ce9f995cfd28/"
+        "sequencing_platform": "/platform-terms/747dc8bf-9e8c-3dbc-967f-ce9f995cfd28/",
+        "seqspec": "/configuration-files/IGVFFI0000CONF/"
```
Comment (gabdank) [inline src/igvfd/tests/data/inserts/sequence_file.json:20]: not sure where is IGVFFI0000CONF file?

**Review by gabdank — CHANGES_REQUESTED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**